packages/protocol/contracts/team/airdrop/ERC20Airdrop2.sol

```diff
@@ -1,129 +0,0 @@
-// SPDX-License-Identifier: MIT
-pragma solidity 0.8.24;
-
-import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
```

Comment:
deleted because we probably won't need it, plus there is this bug: https://github.com/code-423n4/2024-03-taiko-findings/blob/main/data/Shield-Q.md#l-23-user-can-not-claim-the-same-amount-twice-using-the-erc20airdropclaimfunction